spec/Cabin/Seat.vspec

```diff
@@ -28,6 +28,13 @@ IsOccupied:
   type: sensor
   description: Does the seat have a passenger in it.
 
+OccupancyStatus:
+  datatype: string
+  type: sensor
+  allowed: ['Unknown', 'Occupied', 'Empty']
+  description: Occupancy status of the seat.
```

Comment:
And here I believe we talk about a human? Or is a set occupied also if there is a jacket, a bag, a dog or a gorilla in the seat. Or si that up to the OEM to decide?

Reply:
MoM:

- Melissa: Keep definition as OEM specific
- Sunil: Signal interesting for Toyota
- Erik: Please compare with IsOccupied above, do not like us to have both
- Melissa: prefer that there is an unknown value
- Sebastian: In VSS we often do not modify "undefined/unknown"

**Daniel:**

You can model both,

1) If the seat is occupied 
2) the list of things that are detected to be on the seat by whatever mean

I don’t see a problem with that.

**Stefan:**

ISO 7000 https://www.iso.org/obp/ui#iso:grs:7000:2960

in car monitoring is mandatory in Europe (and can be done by camera) [Euro NCAP requirements | VDA](https://www.vda.de/en/topics/automotive-industry/standardization-and-technical-standards/euro-ncap-anforderungen)

Reply:
MoM:

- Decision: OK to keep "unknown"